ouroboros-network/test/Test/Ouroboros/Network/PeerSelection/PeerMetric.hs

```diff
@@ -293,6 +294,104 @@ prop_insert_peer script =
                        joinedAt
         size = Map.size m
 
+-- | #4092
+--
+prop_unit_4092 :: Property
```

Comment:
I don't think we need this, after all this was just a test failure due to the test logic.